src/System.Resources.ResourceManager/tests/MissingManifestResourceExceptionTests.cs

```diff
@@ -33,13 +33,5 @@ public static void ConstructorWithMessageAndInnerException()
             Assert.Equal(message, mmre.Message);
             Assert.Same(innerException, mmre.InnerException);
         }
-
-        [Fact]
-        public void Serialization()
-        {
-            const string message = "FATAL ERROR: The pizza could not be found.";
-            var ex = new MissingManifestResourceException(message);
```

Comment:
No. If type B derives from A and B isn't marked [Serializable], it doesn't matter whether A is; B can't be serialized with BinaryFormatter.